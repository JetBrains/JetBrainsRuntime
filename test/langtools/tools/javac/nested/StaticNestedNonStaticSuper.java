/*
 * @test /nodynamiccopyright/
 * @bug 8291154
 * @summary Disallow static nested subclasses of non-static nested classes
 * @compile/fail/ref=StaticNestedNonStaticSuper.out -XDrawDiagnostics StaticNestedNonStaticSuper.java
 */

class StaticNestedNonStaticSuper{
    public abstract class NonStaticNested {
        public static class StaticNested extends NonStaticNested {
        }
    }
}
