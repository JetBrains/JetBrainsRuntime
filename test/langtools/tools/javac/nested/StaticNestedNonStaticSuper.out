StaticNestedNonStaticSuper.java:10:23: compiler.err.no.encl.instance.of.type.in.scope: StaticNestedNonStaticSuper
1 error
